lib/shell/shellscript.js

```diff
@@ -74,6 +77,18 @@ class ShellScriptWorker {
             } else {
                 console.log('No content type information file found');
             }
+
+            if (await fs.pathExists(optionsFile)) {
+                try {
+                    const options = await fs.readJSON(optionsFile);
+                    if (options.postProcess === true) {
```

Comment:
ptions.postProcess === true || ptions.postProcess === 'true' 

In case the boolean ever gest parsed as string.